docs/translating/chinese.md

```diff
@@ -18,13 +18,13 @@ Zulip 的文风比较口语化，考虑到大多数中国用户的习惯，翻
 
 "Message" can be literally translated as "消息" and "信息", both
 OK. Here "消息" is chosen for translation. For example, "Stream
-Message" is translated as "频道消息", while "Private Message" is
-translated as "私信". The domestic WeiBo, WeChat also keep in line
+Message" is translated as "频道消息", while "Direct Message" is
+translated as "直信". The domestic WeiBo, WeChat also keep in line
 with the habit. "Starred Message" is similar to "Star Mail (星标邮件)"
 feature in QQ Mail, so it is translated into "星标消息".
 
 Message 可直译为“消息”、“信息”等，两者皆可，这里统一选用“消息”。例如，
-“Stream Message”译作“频道消息”；但“Private Message”又译为“私信"，与国
+“Stream Message”译作“频道消息”；但“Direct Message”又译为“直信"，与国
```

Comment:
@chrisbobbe can you help verify this is correct for Chinese? I see this values in our translation files:
```
locale/zh_Hans/translations.json:  "Direct message": "私聊消息",
locale/zh_Hans/translations.json:  "Direct message to me": "",
locale/zh_Hans/translations.json:  "Direct messages": "私信",
```

Reply:
Richard says:

> I personally wouldn't change the translation since the meaning does not change. I didn't like the translation provided but it seems that Microsoft's translation is using the same.